pkg/configs/advisory/v2/document.go

```diff
@@ -131,13 +131,37 @@ var (
 
 // Get returns the advisory with the given ID. If such an advisory does not
 // exist, the second return value will be false; otherwise it will be true.
-func (advs Advisories) Get(id string) (Advisory, bool) {
+func (advs Advisories) Get(id string, aliases []string) (Advisory, bool) {
```

Comment:
I don't think we need to change this function — we already have `GetByVulnerability`, where the input can be the ID or any alias, right?

Reply:
removed that extra parameter